README.md

```diff
@@ -1,2 +1,15 @@
 # OAS Shared Fragments
-This repository contains Spine's API fragments that can be shared amongst other API's OAS spec.
+This repository contains shared fragments for the NHSD API specifications.
+
+### Instruction
+Locate the file that you want to reference and copy the url. Note that the reference should 
+point to the **raw** file:
+
+![raw button](https://raw.githubusercontent.com/NHSDigital/oas-shared-fragments/master/doc/raw.png)
```

Comment:
Doesn't look like the raw button is rendering as expected: https://github.com/NHSDigital/oas-shared-fragments/tree/AMB-473-add-spine-core-errors-fragments